File: JavaCoreGoIt/src/module02/practice20160820/Task03.java
package module02.practice20160820;

public class Task03 {

    static double testWithdraw(double bal, double withdraw, double tax){
        return bal - tax - withdraw;
    }

    public static void main(String[] args) {
        double balance = 250.0;
        double cash = 199.0;
        double tax = 1.5;

        double result = testWithdraw(balance,cash,tax);

        if (result>=0) System.out.println("GO: " + result);
        else System.out.println("Oops..");

    }
}

File: JavaCoreGoIt/z_kotlin/src/module02/practice20160820/Task03.kt
package module02.practice20160820

internal fun testWithdraw(bal: Double, withdraw: Double, tax: Double): Double {
    return bal - tax - withdraw
}

fun main(args: Array<String>) {
    val balance = 200.0
    val cash = 195.0
    val tax = 1.5

    val result = testWithdraw(balance, cash, tax)

    if (result>=0)
        println("GO: "+ result)
    else
        println("Oops..")

}